Dockerfile-local

```diff
@@ -3,6 +3,8 @@ FROM ruby:2.3.1
 RUN apt-get update 
 RUN apt-get install -y --no-install-recommends apt-utils
 RUN apt-get install -y locales
+RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
+RUN apt-get install -y nodejs
 RUN apt-get clean
```

Comment:
This line is funny, because I think this actually makes the image larger instead of smaller.
But let's not save the world when it does not need to be saved.

Reply:
how could it make the image larger?